src/dsnotificationinterface.h

```diff
@@ -16,7 +16,7 @@ class CDSNotificationInterface : public CValidationInterface
 
 protected:
     // CValidationInterface
-    void UpdatedBlockTip(const CBlockIndex *pindex);
+    void UpdatedBlockTip(const CBlockIndex *pindexNew, const CBlockIndex *pindexFork, bool fInitialDownload);
```

Comment:
Good idea 👍 I'll include this in the next one to keep cleanup/refactoring separate from bug fixing.